WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -185,6 +186,13 @@ class PostCardStatusViewModel: NSObject {
                 buttons.append(.share)
             }
 
+            let canBlaze = post.blog.isBlazeApproved &&
+                post.status == .publish &&
+                post.password == nil
```

Comment:
[Nit] Add this as a computed variable in `AbstarctPost.swift`

Reply:
Fixed! [d3d7f34](https://github.com/wordpress-mobile/WordPress-iOS/pull/20135/commits/d3d7f34d39692b4297a78d128e5e47c264e9afc3)